### PR TITLE
`<BrowserPreviewWidget/>`- new component

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
New component- BrowserPreviewWidget.
Related Zeplin file: https://zpl.io/2jMDQLr

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

File: src/BrowserPreviewWidget/BrowserPreviewWidget.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { browserBarSizes, dataHooks } from './constants';
+import PreviewWidget from '../PreviewWidget';
+import styles from './BrowserPreviewWidget.st.css';
+
+/** Browser preview widget */
+class BrowserPreviewWidget extends React.PureComponent {
+  static displayName = 'BrowserPreviewWidget';
+
+  static propTypes = {
+    /** Browser preview widget data hook*/
```
Comment (mosheKerbel) [inline src/BrowserPreviewWidget/BrowserPreviewWidget.js]:
```suggestion
    /** Applied as data-hook HTML attribute that can be used in the tests*/
```

File: src/BrowserPreviewWidget/BrowserPreviewWidget.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    dataHook: string,
+
+    /** Background skin. To use `custom` skin, set it to `custom` and use the `backgroundColor` prop*/
+    skin: oneOf(['neutral', 'gradient', 'custom']),
+
+    /** Mobile preview widget background color. Can be set with `design system` colors*/
```
Comment (mosheKerbel) [inline src/BrowserPreviewWidget/BrowserPreviewWidget.js]: hmm what are design system colors?

File: src/BrowserPreviewWidget/BrowserPreviewWidget.uni.driver.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { baseUniDriverFactory } from 'wix-ui-test-utils/base-driver';
+
+export const browserPreviewWidgetDriverFactory = base => {
+  return {
+    ...baseUniDriverFactory(base),
+
+    /** Get the current count */
+    getCountText: async () =>
```
Comment (mosheKerbel) [inline src/BrowserPreviewWidget/BrowserPreviewWidget.uni.driver.js]: is this an auto generated code?

File: src/BrowserPreviewWidget/test/BrowserPreviewWidget.spec.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  const render = createRendererWithUniDriver(
+    browserPreviewWidgetPrivateDriverFactory,
+  );
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('should render', async () => {
+    const { driver } = render(<BrowserPreviewWidget />);
```
Comment (mosheKerbel) [inline src/BrowserPreviewWidget/test/BrowserPreviewWidget.spec.js]: is this an auto generated code?

File: src/BrowserPreviewWidget/test/BrowserPreviewWidget.visual.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import BrowserPreviewWidget from '../BrowserPreviewWidget';
+
+const commonProps = {
+  //use for repeated props across the tests (e.g. {buttonText: 'example'})
+};
+
+const tests = [
```
Comment (mosheKerbel) [inline src/BrowserPreviewWidget/test/BrowserPreviewWidget.visual.js]: i'd expect to see visual tests for the different api values of browser size etc'...

File: src/BrowserPreviewWidget/BrowserPreviewWidget.js (hunk 2 above)
Comment (TalZa1810) [reply src/BrowserPreviewWidget/BrowserPreviewWidget.js]: I talked to Nitay about it. We refer to `D80` etc... Do you have a suggestion for a different description?

**Comment by TalZa1810** [comment]
I've added a table with a description which browser bar height should be used with each width. Is it clear enough? WDYT? 

Should I merge this after UX review?

https://wix-wix-style-react-pr-4515.surge.sh/?selectedKind=WIP&selectedStory=BrowserPreviewWidget&full=0&addons=0&stories=1&panelRight=0

File: src/BrowserPreviewWidget/BrowserPreviewWidget.js (hunk 2 above)
Comment (mosheKerbel) [reply src/BrowserPreviewWidget/BrowserPreviewWidget.js]: i see in the code that it falls back to normal color if not exists in wsr color, so i think it's fine...

**Comment by mosheKerbel** [comment]
@TalZa1810 notice that u have conflicts

**Comment by mosheKerbel** [comment]
> I've added a table with a description which browser bar height should be used with each width. Is it clear enough? WDYT?
> 
> Should I merge this after UX review?
> 
> https://wix-wix-style-react-pr-4515.surge.sh/?selectedKind=WIP&selectedStory=BrowserPreviewWidget&full=0&addons=0&stories=1&panelRight=0

nice looks really informative!

**Pull request merged**